src/homepageExperience/components/ConnectionInformation.tsx

```diff
@@ -0,0 +1,75 @@
+// Libraries
+import React, {PureComponent} from 'react'
+
+// Decorator
+import {ErrorHandling} from 'src/shared/decorators/errors'
+
+export enum LoadingState {
+  NotStarted = 'NotStarted',
+  Loading = 'Loading',
+  Done = 'Done',
+  NotFound = 'NotFound',
+  Error = 'Error',
+}
+
+export interface Props {
+  loading: LoadingState
+  bucket: string
+  countDownSeconds: number
+}
+
+@ErrorHandling
+class ConnectionInformation extends PureComponent<Props> {
+  public render() {
+    return (
+      <div data-testid="connection-information">
+        <h4 className={`wizard-step--text-state ${this.className}`}>
+          {this.header}
+        </h4>
+        <p>{this.additionalText}</p>
+      </div>
+    )
+  }
+
+  private get className(): string {
+    switch (this.props.loading) {
+      case LoadingState.Loading:
+        return 'loading'
+      case LoadingState.Done:
+        return 'success'
+      case LoadingState.NotFound:
+      case LoadingState.Error:
+        return 'error'
+    }
+  }
+
+  private get header(): string {
+    switch (this.props.loading) {
+      case LoadingState.Loading:
+        return 'Awaiting Connection...'
+      case LoadingState.Done:
+        return 'Connection Found!'
+      case LoadingState.NotFound:
+        return 'Data Not Found'
+      case LoadingState.Error:
+        return 'Error Listening for Data'
+    }
+  }
+
+  private get additionalText(): any {
+    const docs = (
+      <span> Make sure the correct bucket is selected and then retry. </span>
```

Comment:
This file was entirely reused, with only this line changed.

Reply:
(It was referencing to telegraf configuration before, I changed it to a more relevant text for the homepage experience.)